.github/workflows/test.yml

```diff
@@ -2,7 +2,6 @@ name: Test
 
 on:
   pull_request:
-    branches: main
```

Comment:
We want tests and linter to run on _any_ PR's, not just the ones based off of `main`.